base_rest/views/base_rest_view.xml

```diff
@@ -2,11 +2,11 @@
 <!-- Copyright 2018 ACSONE SA/NV
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
-
     <record id="menu_rest_api_root" model="ir.ui.menu">
         <field name="name">REST API</field>
         <field name="sequence" eval="400" />
         <field name="web_icon">base_rest,static/description/icon.png</field>
+        <field name="groups_id" eval="[(4, ref('base.group_system'))]" />
```

Comment:
@aleuffre Thank you for this proposal. Why not... but something must be set into the readme to describe how you can get access to the REST api documentation.